cmd/skopeo/layers.go

```diff
@@ -73,7 +73,7 @@ var layersCmd = cli.Command{
 			if !strings.HasPrefix(digest, "sha256:") {
 				digest = "sha256:" + digest
 			}
-			r, blobSize, err := rawSource.GetBlob(digest)
+			r, blobSize, err := rawSource.GetBlob(types.BlobInfo{Digest: digest})
```

Comment:
This should probably set `Size` to -1 (unknown), depending on how #147 defines the `GetBlob` requirements.